src/Subject.ts

```diff
@@ -5,7 +5,7 @@ import { Subscriber } from './Subscriber';
 import { ISubscription, Subscription, TeardownLogic } from './Subscription';
 import { ObjectUnsubscribedError } from './util/ObjectUnsubscribedError';
 import { SubjectSubscription } from './SubjectSubscription';
-import { $$rxSubscriber } from './symbol/rxSubscriber';
+import { rxSubscriber as Symbol_rxSubscriber } from './symbol/rxSubscriber';
```

Comment:
@DzmitryShylovich oh I meant that `Symbol.observable` -> `Symbol_observable`, `Symbol.iterator` -> `Symbol_iterator` ... those are polyfills.  `rxSubscriber` is just an RxJS library-specific symbol. It wouldn't exist on `window.Symbol` or the like.